tests/CMakeLists.txt

```diff
@@ -35,7 +35,9 @@ if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
             add_subdirectory(cppException)
             add_subdirectory(ecall)
             add_subdirectory(file)
-            add_subdirectory(mbed)
+            if (NOT OE_PREFIX_MBEDTLS_SYMBOLS)
```

Comment:
If the user scenario is that a dev can clone our repo and rebuild using this flag so they can quarantine our use of mbedtls, then there should be an expectation that the resulting mbedtls library is tested as well. Can we modify the tests/mbed build to also do the symbol rename if OE_PREFIX_MBEDTLS_SYMBOLS?